src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -304,6 +308,24 @@ internal static ArgumentOutOfRangeException InvalidCommandType(CommandType value
             return InvalidEnumerationValue(typeof(CommandType), (int)value);
         }
 
+        internal static ArgumentOutOfRangeException InvalidCommandBehavior(CommandBehavior value)
+        {
+#if DEBUG
+            if ((0 <= (int)value) && ((int)value <= 0x3F))
+            {
+                Debug.Assert(false, "valid CommandType " + value.ToString());
+            }
+#endif
```

Comment:
Looks like InvalidCommandBehavior is only referenced by ValidateCommandBehavior, so I can just combine the two.